jquery.sumoselect.js

```diff
@@ -793,6 +793,7 @@
                     var O = this;
                     O.createElems();
                     O.setText();
+                    setTimeout(function(){ O.E.trigger('sumo:rendered'); }, 100);

```

Comment:
Using `setTimeout` here doesn't help anyone, since the actual time everything gets rendereddoesn't take `100ms` for everyone.